### PR TITLE
Add an orientation page about Create your application by type

### PR DESCRIPTION
**Pull request by mere-teresa**
Choices are:
* frontend only
* backend without users management
* backend with users managmenent

### CODE REVIEW

File: doc_sphinx/tutorials/2_app_choice.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+=============================
+
+The implementation of your app depends on your needs.
+You can create several types of applications.
+
+Frontend only application
+==========================
+
+Chose this way if you are developping a frontend-only application, ie an app for tablet or mobile.
+In addition to that, maybe that your main programming language is Javascript.
```
Comment (Amoki) [inline doc_sphinx/tutorials/2_app_choice.rst]: javascript for browser application, but any other language for other purposes (swift for iOS, java for android, etc)

Maybe the concept of backend-less is better than frontend-only.

File: doc_sphinx/tutorials/2_app_choice.rst [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+-------------
+
+Chose this way if your backend app has no Users.
+Note that the application itself is a user.
+=> `Create a backend app`_
+
+With Users
+-------------
+
+Chose this way if your backend application is bound with Users credentials,
```
Comment (Amoki) [inline doc_sphinx/tutorials/2_app_choice.rst]: I'd put the "with users" before the "without".

"Chose this way if your backend application will use BIMData Connect users. Therefore, these users will be able to share their data between multiple app (between your app and BIMData Platform by example)"

File: doc_sphinx/tutorials/2_app_choice.rst [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+In addition to that, maybe that your main programming language is Javascript.
+=> `Create a frontend app`_
+
+Application with a backend
+===========================
+
+Without Users
+-------------
+
+Chose this way if your backend app has no Users.
```
Comment (Amoki) [inline doc_sphinx/tutorials/2_app_choice.rst]: Chose this way if you don't want to use BIMData Connect users. 
You'll have to handle authorization access yourself and will not be able to share data with other BIMData applications (eg: BIMData Platform), but basic setup is easier.

File: doc_sphinx/tutorials/2_app_choice.rst (hunk 1 above)
Comment (mere-teresa) [reply doc_sphinx/tutorials/2_app_choice.rst]: I am currently investigating if there is a word "no backend" or "backend-less" to describe it.

**Pull request merged**